host/xtest/pkcs11_1000.c

```diff
@@ -8717,10 +8717,12 @@ static void xtest_pkcs11_test_1027(ADBG_Case_t *c)
 	if (!ADBG_EXPECT_CK_OK(c, rv))
 		goto out_unsetenv;
 
+	Do_ADBG_BeginSubCase(c, "Set ACL TEE client identity");
```

Comment:
This is not really  a sub case of  test 1027. This sets the group identity for user login tests performed in `test_login_logout_acl_auth()`.

Reply:
The comment has been addressed